src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
+  userId: number;
```

Comment:
Since the userId property is not used in the task, you can consider removing it from the Todo interface to keep the code clean.

Reply:
done